### PR TITLE
Webpack todo checkbox

### PR DESCRIPTION
**Pull request by abiy006**
Tasks accomplished on this PR:

- Checkbox is added
- Checkbox click event is added
- Checkbox click event action is added to the local storage

### CODE REVIEW

File: src/checkbox_todo.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  todos[index].completed = !todos[index].completed;
+  localStorage.setItem('todos', JSON.stringify(todos));
+}
+
+export const clearCompletedChkBox = (todos) => {
+  todos.forEach((todo, index) => {
+    if (todo.completed) {
+      todos.splice(index, 1);
+    }
+  });
```
Comment (mckent05) [inline src/checkbox_todo.js]: - Good job done so far, according to the project requirements, when the `clear all button` is clicked, it should delete all completed todos. But this isn't so in your project as it deletes all to-dos except one. Please see the image below. I suggest using the `Array.filter` method to filter all completed todos 👍 👍 

Your project(after clicking on the completed todos):
wwrgg

Your project(When I click on clear all button):
rrgwr

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @abiy006, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- You did well updating the local storage with the updated array of todo 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,160 @@
-# webpack-todo
\ No newline at end of file
+# Portfolio
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="Images\html_css.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (V-Blaze) [inline README.md:6]: - [OPTIONAL] I'll suggest you remove this Microverse logo as it was placed here as a placeholder. The image is currently broken and it makes your readme look a bit unprofessional. :+1: :smile:

**Review by V-Blaze — APPROVED** [review]
Hi @abiy006  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Toggle completed status works great💯
-  Clear completed works fine👍
-  Divided code into modules🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**